apps/pipelines/nodes/nodes.py

```diff
@@ -56,9 +56,23 @@
 from apps.utils.prompt import OcsPromptTemplate, PromptVars, validate_prompt_variables
 
 OptionalInt = Annotated[int | None, BeforeValidator(lambda x: None if x == "" else x)]
+DEFAULT_TAG = "The tag that the output message should be tagged with"
 
 
-class RenderTemplate(PipelineNode):
+class TagMixin(BaseModel):
+    tag: str = Field(
+        default=DEFAULT_TAG,
+        json_schema_extra=UiSchema(widget=Widgets.expandable_text),
+    )
```

Comment:
suggested field changes:
```suggestion
    tag: str = Field(
        default="",
        title="Message Tag",
        description="The tag that the output message should be tagged with",
    )
```

Reply:
> tag: str = Field(

Tried this. description is not visible as help text inside or outside the field.

Reply:
You need to remove the tag field from the `LlmResponseWithPrompt` node since it's overriding the one in the mixin.

Reply:
done